tests/conftest.py

```diff
@@ -123,6 +123,22 @@ def url_all_origins(request):
     return request.param
 
 
+# Reads the given number of messages from the websocket, and returns them
+# in consistent order.
+@pytest.fixture
+def read_sorted_messages(websocket):
+
+    async def _(message_count):
```

Comment:
You can call the inner function `read_sorted_messages` as well (I've seen this pattern in WPT fixtures IIRC)

Reply:
done